base/logging.cc

```diff
@@ -984,7 +984,10 @@ LogMessage::~LogMessage() {
     //   free_something();
     // }
 #if defined(STARBOARD)
-    SbLog(LogLevelToStarboardLogPriority(severity_), str_newline.c_str());
+    if ((g_logging_destination & LOG_TO_SYSTEM_DEBUG_LOG) == 0) {
```

Comment:
Can this conditional instead be part of ShouldLogToStderr?

Reply:
Yes, good idea.